bot.go

```diff
@@ -57,15 +60,50 @@ func handleDownload(b *bot.Bot, m *bot.Message) {
 }
 
 func handleSearch(b *bot.Bot, m *bot.Message, api *rarbg.API) {
-	re, _ := regexp.Compile("tt[0-9]{7}") // e.g. tt0137523
-	imdbIDs := re.FindAllString(m.Text, -1)
+	imdbIDs, err := searchIMDbIDsFromMessage(m.Text)
+	if err != nil {
+		b.Send(m.Sender, replyNoIMDbIDS+err.Error())
+		return
+	}
 	if len(imdbIDs) == 0 {
 		b.Send(m.Sender, helpText)
 		return
 	}
+
 	for _, id := range imdbIDs {
 		result := new(bytes.Buffer)
 		searchIMDb(result, id, api)
 		b.Send(m.Sender, result.String(), &bot.SendOptions{ParseMode: bot.ModeMarkdown})
 	}
 }
+
+func searchIMDbIDsFromMessage(text string) ([]string, error) {
+	imdbIDs := []string{}
+	// Douban
+	movieLinks := findDoubanMovieURLs(text)
+	for _, url := range movieLinks {
+		movie := douban.NewMovie()
+		if err := movie.FetchFromURL(url); err != nil {
+			return nil, err
```

Comment:
How about not return in a rush. I feel like to

```
log.Printf("error while fetch from url %s: %s", url, err)
continue
```

Reply:
You mean the error shouldn't be noticed by user?

Reply:
Hmm, I thought too much for multiple links, nvm.